spec/json/tsg_email_validation_v3.json

```
@@ -225,10 +225,427 @@
+                ],
+                "example": {
+                  "file_type": "csv"
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "The request was successful. The response contains the URI and headers where you will upload your email address list.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/PutValidationsEmailJobs200Response"
+                },
+                "examples": {
+                  "response": {
+                    "value": {
+                      "job_id": "01H793APATD899ESMY25ZNPNCF",
+                      "upload_uri": "https://example.com/",
+                      "upload_headers": [
+                        {
+                          "header": "x-amz-server-side-encryption",
+                          "value": "aws:kms"
+                        },
+                        {
+                          "header": "content-type",
+                          "value": "text/csv"
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request. Invalid or missing `file_type` or malformed request.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ErrorResponse"
+                },
+                "examples": {
+                  "response": {
+                    "value": {
+                      "errors": [
+                        {
+                          "message": "Error message"
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ErrorResponse"
+                },
+                "examples": {
+                  "response": {
+                    "value": {
+                      "errors": [
+                        {
+                          "message": "error message"
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "get": {
+        "operationId": "GetValidationsEmailJobs",
+        "summary": "This request lists all of a User's Bulk Email Validation Jobs.",
+        "tags": [
+          "Bulk Email Address Validation"
+        ],
+        "description": "**This endpoint returns a list of all of your Bulk Email Validation Jobs.**\n",
+        "responses": {
+          "200": {
+            "description": "Success",
```

```suggestion
            "description": "The request was successful. The response contains a list of all of your Bulk Email Validation Jobs."
```
If detailed description is helpful